### PR TITLE
Fixed source-tag info

### PR DESCRIPTION
**Pull request by kz6fittycent**
- Line 41:  source-tag: v$SNAPCRAFT_PROJECT_VERSION
  - Changed version (in `yaml`) info to match snap that was built
My apologies @merlijn-sebrechts - I missed that bit about `source-tag`, as I was focused on other items.

### CODE REVIEW

**Review by merlijn-sebrechts — APPROVED** [review]
LGTM

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -38,6 +41,7 @@ parts:
   jenkins:
     plugin: nil
     source: https://github.com/jenkinsci/jenkins.git
+    source-tag: $SNAPCRAFT_PROJECT_VERSION
```
Comment (merlijn-sebrechts) [inline snap/snapcraft.yaml]: The tags in the upstream repo are prefixed with `jenkins-`

```suggestion
    source-tag: jenkins-$SNAPCRAFT_PROJECT_VERSION    
```

See https://github.com/jenkinsci/jenkins/tags

**Review by merlijn-sebrechts — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (kz6fittycent) [reply snap/snapcraft.yaml]: Thanks @merlijn-sebrechts - I appreciate your patience w/ me as I learn these workflows and their ins 'n outs. I think, maybe, this time, it's good.

Comment (merlijn-sebrechts) [reply snap/snapcraft.yaml]: Don't beat yourself up about it! Everyone makes mistakes.

**Review by merlijn-sebrechts — APPROVED** [review]
Check succeeded so we should be good to go! Let's test this!

**Comment by kz6fittycent** [comment]
fingers crossed

**Pull request merged**